code/__DEFINES/economy.dm

```diff
@@ -65,6 +65,9 @@
 #define CIV_JOB_GROW 12
 #define CIV_JOB_ATMOS 13
 #define CIV_JOB_RANDOM 14
+//BUBBER EDIT START
+#define CIV_JOB_SMITH 21 //By making this higher we avoid having to maintain this value if more bounties are added upstream
```

Comment:
For future reference by anyone else this works just as well technically if you do it modularly but you should NOT do this modularly, as these defines need to be unique, and if they're not you get awful subtle bugs.